facefusion/processors/modules/face_editor.py

```diff
@@ -205,26 +201,16 @@ def apply_edit_face(crop_vision_frame : VisionFrame, face_landmark_68 : FaceLand
 		})[0]
 
 	with thread_semaphore():
-		motion_points = motion_extractor.run(None,
+		rotation, scale, translation, expression, motion_points_edit, motion_points = motion_extractor.run(None,
```

Comment:
the inputs is named angle, scale, translation, expression, motions_points_raw, motion_points - please follow that naming

Reply:
it is not really angle, but a rotation matrix

Reply:
then we have to rename it to rotation_matrix